app/models/facility.rb

```diff
@@ -1,5 +1,6 @@
 class Facility < ApplicationRecord
-  belongs_to :district
+  belongs_to :district, inverse_of: :facilities
+  belongs_to :user
```

Comment:
```suggestion
  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

`belongs_to :user` threw me off.